### ISSUE TITLE
Comment count number text fails contrast accessibility

### ISSUE THREAD

**Issue by HarryFischer** [guardian/dotcom-rendering#4731]
Icon and text should be the same colour
New colour to follow

Current: 
![image](https://user-images.githubusercontent.com/20658471/165287174-0a23a34b-96ba-4def-bb35-c04c1dc00f09.png)

**Comment by Bryony-Szekeres** [comment]
All metadata (timestamp, comment icon and comment count) on cards should conform to the following colour logic:

- On cards with pale backgrounds, the metadata is always neutral.46
- On cards with dark backgrounds (ie media, special report) the metadata is always neutral.86
- On cards in solid pillar colours (ie live blog cards) the metadata is always neutral.100 (other than Labs where it is neutral.7)
- On <event> <eventAlt> and <longRunningAlt> fronts palettes cards the metadata is #606060

This Figma layout shows all the colours as they are now and what they should be:
https://www.figma.com/proto/5CfbWOeZPRi15lXBD5u1rW/Card-layout-system?page-id=0%3A1&node-id=45%3A43534&viewport=-12253%2C897%2C0.39&scaling=min-zoom

**Comment by alinaboghiu** [comment]
I think these were done, please reopen if not, I haven't checked every single one
